tad/schema/system_card.py

```diff
@@ -0,0 +1,12 @@
+from pydantic import (
+    BaseModel,
+    Field,  # pyright: ignore [reportUnknownMemberType]
+)
+
+from tad.schema.instrument import InstrumentBase
+
+
+class SystemCard(BaseModel):
+    schema_version: str = Field(default="0.1a6")
```

Comment:
what is this default based on and is it wise to have a default at all?

Reply:
Default is based on what is currenlty in the repo. I needed to add something because the field is required and i did not want to ask the user